### PR TITLE
Laurent issue#2266 shup reimport if status ERROR

### PR DESCRIPTION
**Pull request by LaurentPV**
Allow import function in case previous try was in ERROR.

Bugfix in case of qualityControlResult failed Valid and no updatedSubjectStudy is returned. In this case for now no update of the quality tag is made on server side so if previously the subjectStudy has tag VALID it will stay like that.

Bugfix of misplaced delete and deleteAll buttons tooltips

### CODE REVIEW

File: shanoir-uploader/src/main/java/org/shanoir/uploader/nominativeData/CurrentNominativeDataController.java [hunk 1]
```diff
@@ -44,20 +44,22 @@ public class CurrentNominativeDataController {
...
+	@SuppressWarnings("deprecation")
 	public void configure(final File workFolderFilePath, final CurrentUploadsWindowTable cuw) {
 		this.currentNominativeDataModel = new CurrentNominativeDataModel();
 		currentNominativeDataModel.addObserver(cuw);
 		this.cuw = cuw;
 		processWorkFolder(workFolderFilePath);
 
-		cuw.table.addMouseListener(new MouseAdapter() {
-			public DefaultTableModel model = (DefaultTableModel) cuw.table.getModel();
+		CurrentUploadsWindowTable.table.addMouseListener(new MouseAdapter() {
```
Comment (michaelkain) [inline shanoir-uploader/src/main/java/org/shanoir/uploader/nominativeData/CurrentNominativeDataController.java:54]: good! I would remove the cuw attribute, when you pass directly by the static attribute of CUWT?

Comment (michaelkain) [reply shanoir-uploader/src/main/java/org/shanoir/uploader/nominativeData/CurrentNominativeDataController.java:54]: even remove the param from configure

**Review by michaelkain — COMMENTED** [review]
Excellent, my local tests are all fine, code works.
Please adapt my minor changes requests and then we merge.
With kind regards, Michael

**Comment by michaelkain** [comment]
Good job Laurent!

**Comment by michaelkain** [comment]
Hi @LaurentPV, as soon as you integrate all my comments, as soon I merge :-)

File: shanoir-uploader/src/main/resources/messages_fr.properties [hunk 2]
```diff
@@ -112,8 +112,8 @@ shanoir.uploader.editPanel.anonymisedLabel=D\u00E9j\u00E0 anonymis\u00E9?:
 shanoir.uploader.exportimport=Exporter les donn\u00E9es
 shanoir.uploader.exportonly=Export uniquement
 shanoir.uploader.existingSubjectsLabel=OU choisir un sujet existant :
-shanoir.uploader.failedUploadsMessagePart1=<html><pre>Une erreur est survenue\n pendant le t\u00E9l\u00E9chargement
-shanoir.uploader.failedUploadsMessagePart2=\nSupprimez le t\u00E9l\u00E9chargement\n qui a \u00E9chou\u00E9 et essayez\n de le t\u00E9l\u00E9charger \u00E0 nouveau</pre></html>
+shanoir.uploader.failedUploadsMessagePart1=<html><pre>Une erreur est survenue \npendant le t\u00E9l\u00E9chargement
+shanoir.uploader.failedUploadsMessagePart2=\nVous pouvez tenter de lancer \nà nouveau le t\u00E9l\u00E9chargement \nen cliquant sur "R\u00E9importer".</pre></html>
```
[the remark this answers is not in the document]
Comment (LaurentPV) [reply shanoir-uploader/src/main/resources/messages_fr.properties:116]: Sorry I missed that. I changed my mind about a "Reimport" button during development. keeping a functional "Import" button seemed more simple and clear finally.

File: shanoir-uploader/src/main/java/org/shanoir/uploader/nominativeData/CurrentNominativeDataController.java (hunk 1 above)
Comment (LaurentPV) [reply shanoir-uploader/src/main/java/org/shanoir/uploader/nominativeData/CurrentNominativeDataController.java]: I modified the instantiation of cut as a true singleton loaded once and for all by ReadyState.load()

**Review by michaelkain — APPROVED** [review]

**Pull request merged**